drizzlepac/haputils/svm_quality_graphics.py

```diff
@@ -126,6 +127,56 @@ def build_svm_plots(data_source, output_basename='', display_plot=False):
+                            'Interfilter_cross-matched_xcentroid_ref_comparison_-_reference_separation_statistics.Non-clipped_median': 'x_non-clipped_median',
+                            'Interfilter_cross-matched_xcentroid_ref_comparison_-_reference_separation_statistics.Non-clipped_standard_deviation': 'x_non-clipped_rms',
+                            'Interfilter_cross-matched_xcentroid_ref_comparison_-_reference_separation_statistics.3x3_sigma-clipped_mean': 'x_3x3_sigma-clipped_mean',
+                            'Interfilter_cross-matched_xcentroid_ref_comparison_-_reference_separation_statistics.3x3_sigma-clipped_median': 'x_3x3_sigma-clipped_median',
+                            'Interfilter_cross-matched_xcentroid_ref_comparison_-_reference_separation_statistics.3x3_sigma-clipped_standard_deviation': 'x_3x3_sigma-clipped_rms',
+                            'Interfilter_cross-matched_ycentroid_ref_comparison_-_reference_separation_statistics.Non-clipped_min': 'y_non-clipped_min',
+                            'Interfilter_cross-matched_ycentroid_ref_comparison_-_reference_separation_statistics.Non-clipped_max': 'y_non-clipped_max',
+                            'Interfilter_cross-matched_ycentroid_ref_comparison_-_reference_separation_statistics.Non-clipped_mean': 'y_non-clipped_mean',
+                            'Interfilter_cross-matched_ycentroid_ref_comparison_-_reference_separation_statistics.Non-clipped_median': 'y_non-clipped_median',
+                            'Interfilter_cross-matched_ycentroid_ref_comparison_-_reference_separation_statistics.Non-clipped_standard_deviation': 'y_non-clipped_rms',
+                            'Interfilter_cross-matched_ycentroid_ref_comparison_-_reference_separation_statistics.3x3_sigma-clipped_mean': 'y_3x3_sigma-clipped_mean',
+                            'Interfilter_cross-matched_ycentroid_ref_comparison_-_reference_separation_statistics.3x3_sigma-clipped_median': 'y_3x3_sigma-clipped_median',
+                            'Interfilter_cross-matched_ycentroid_ref_comparison_-_reference_separation_statistics.3x3_sigma-clipped_standard_deviation': 'y_3x3_sigma-clipped_rms',
+                            'Interfilter_cross-matched_comparison_-_reference_separations.delta_xcentroid_ref': 'comp-ref_x_seperation',
+                            'Interfilter_cross-matched_comparison_-_reference_separations.delta_ycentroid_ref': 'comp-ref_y_seperation'}
+
+    # Get the requested columns from the dataframe in addition columns
+    # added by the pandas_utils
+    if_xm_DF = get_pandas_data(data_source, intfilt_xm_col_names)
+
+    # Rename the columns to abbreviated text for ease of management
+    for old_col_name, new_col_name in intfilt_xm_col_names.items():
+        if_xm_DF.rename(columns={old_col_name: new_col_name}, inplace=True)
+
+    # remove rows that aren't relevant
+    if_xm_DF = if_xm_DF[np.isnan(if_xm_DF.ref_image_platescale) == False]
+
```

Comment:
If there are no rows in `if_xm_DF`, then it should not even call the plotting routine in order to prevent a crash.  This type of logic needs to be included for all the plots, but we will update the rest of the code in later PRs.

Reply:
Added come code to build_gaia_plots() to add a title text block to top of the svm_qa_gaia_comparison.html file in commit [faac0c8](https://github.com/spacetelescope/drizzlepac/pull/741/commits/faac0c87b2087d849e431e014e1a9ffb87761fa5)

Reply:
Issue resolved in commit [6f74e02](https://github.com/spacetelescope/drizzlepac/pull/741/commits/6f74e02e1daecca8e72cbfe1916801e8e402fc9d)